superset/utils/log.py

```diff
@@ -47,10 +47,12 @@ def collect_request_payload() -> dict[str, Any]:
 
     payload: dict[str, Any] = {
         "path": request.path,
-        **request.form.to_dict(),
-        # url search params can overwrite POST body
-        **request.args.to_dict(),
     }
+    payload.update(**request.form.to_dict())
+    payload.update(**request.args.to_dict())
```

Comment:
```suggestion
```

Reply:
the 2 lines look identical at first glance but they're different. Actually now I'm not sure why I changed lines 50-52... Let me roll this back to what it was before